opm/simulators/flow/BlackoilModelEbos.hpp

```diff
@@ -27,6 +27,24 @@
 #include <ebos/eclproblem.hh>
 #include <opm/models/utils/start.hh>
 
+#include <opm/simulators/timestepping/AdaptiveTimeSteppingEbos.hpp>
+
+#include <opm/simulators/flow/NonlinearSolverEbos.hpp>
+#include <opm/simulators/flow/BlackoilModelParametersEbos.hpp>
+#include <opm/simulators/wells/BlackoilWellModel.hpp>
+#include <opm/simulators/aquifers/BlackoilAquiferModel.hpp>
+#include <opm/simulators/wells/WellConnectionAuxiliaryModule.hpp>
+#include <opm/simulators/flow/partitionCells.hpp>
+#include <opm/simulators/flow/SubDomain.hpp>
+#include <opm/simulators/flow/countGlobalCells.hpp>
+#include <opm/simulators/utils/DeferredLoggingErrorHelpers.hpp>
+#include <opm/simulators/linalg/extractMatrix.hpp>
```

Comment:
Is there a reason for this ordering?  We're mixing `simulators/flow`, `simulators/wells`, `simulators/aquifers`, `simulators/utils`, and `simulators/linalg` which at least superficially draws the eye.